openlibrary/plugins/openlibrary/js/FileValidation.js

```diff
@@ -0,0 +1,29 @@
+export default function Filevalidation (){
```

Comment:
I'd suggest naming this function something more meaningful. Given it's an event handler onFileInputValidation might be a good name.